### PR TITLE
Issue #360: Fixing and improving FlagTemplateParser

### PR DESCRIPTION
**Pull request by Nono314**
Fixed:

- {{flag|...}} template now works with a country code as 1st parameter
- 2-letter codes for NL are now actually supported

Improved:

- IOC codes are now supported along with ISO codes, both as template parameter and as template themselves + a few common non standard codes (UK, EU...)
- template name are now defined per language in config file (eg {{drapeau2|...}} for frwiki or {{bandera2|..}} for eswiki)
- added setup for French country names
- added unit tests for all test cases

### CODE REVIEW

File: core/src/main/scala/org/dbpedia/extraction/config/dataparser/FlagTemplateParserConfig.scala [hunk 1]
```diff
@@ -991,244 +1518,11 @@ object FlagTemplateParserConfig
                 "VNM"->"Vietnam",
                 "WLF"->"Wallis eta Futuna",
                 "YEM"->"Yemen",
-                "ZMB"->"Zambia",
-                "NZL"->"Zeelanda Berria",
-                "ZWE"->"Zimbabwe",
-                "CYP"->"Zipre"
-            ), // eu end
-        "ga" -> // http://en.wikipedia.org/wiki/ISO_3166-1
```
Comment (jimkont) [inline core/src/main/scala/org/dbpedia/extraction/config/dataparser/FlagTemplateParserConfig.scala]: is `ga` removed from the configuration?

**Comment by Nono314** [comment]
It should definitely not have been removed.
This config file has grown quite a bit and my local git client was telling me the diff was too large to display, so I didn't spot the issue. I should however probably have double checked online before sumitting the PR. My bad :(

File: core/src/main/scala/org/dbpedia/extraction/config/dataparser/FlagTemplateParserConfig.scala [hunk 2]
```diff
@@ -748,6 +768,513 @@ object FlagTemplateParserConfig
...
+              "CCK"->"Islas Cocos",
+              "MHL"->"Islas Marshall",
+              "FRO"->"Islas Feroe",
+              "SLB"->"Islas Salomón",
+              "STP"->"Santo Tomé y Príncipe",
+              "ZMB"->"Zambia",
+              "ZWE"->"Zimbabue",
+              "ARE"->"Emiratos Árabes Unidos"
+            ), //end es
+        "fr" ->
```
Comment (jimkont) [inline core/src/main/scala/org/dbpedia/extraction/config/dataparser/FlagTemplateParserConfig.scala]: Can you also put `fr` in alphabetical order? other than that look quite good

Comment (Nono314) [inline core/src/main/scala/org/dbpedia/extraction/config/dataparser/FlagTemplateParserConfig.scala]: There was no actual alphabetical order in this config file, I even pulled `es` from the bottom toward the top... So let's sort them all!

Comment (jimkont) [inline core/src/main/scala/org/dbpedia/extraction/config/dataparser/FlagTemplateParserConfig.scala]: this time the diff was too big for me to notice that :) I saw that you put es in order but then noticed fr

**Comment by jimkont** [comment]
Looks good to me, anyone else?

**Comment by Nono314** [comment]
@jimkont Any feedback about the points at #360 ?

**Comment by jimkont** [comment]
Let's merge this and think about  #360 separately.
Thanks @Nono314

**Pull request merged**